Makefile

```diff
@@ -76,7 +76,7 @@ define ci_setup_helper
 	$(eval build_function := $(strip $(3)))
 	$(eval guard_variable := $(strip $(4)))
 	@# First, if the dependency is installed, we can bail early
-	$(if $(shell bash -c '$(1)'),$(eval $(guard_variable) := true),
+	$(if $(shell '$(1)'),$(eval $(guard_variable) := true),
```

Comment:
if we could just set the SHELL variable to `bash` for the entire Makefile that would be great. I can't even run the qemu setup on my (ubuntu) machine right now because the rule uses bash syntax.

Reply:
This particular change I don't think has broken anything. But when I run `make ci-job-qemu` on master for my machine it fails locally with this error: 
`/bin/sh: 1: Syntax error: word unexpected (expecting ")")`.

I think that this error doesn't happen in CI because when users call `ci-job-qemu` we prompt them about whether they want to do setup, but in CI we just do the setup, so the Makefile rules executed on my machine are not the same as those executed in CI.

But I have run into this issue a couple times now when we run different rules locally vs. in CI.

Reply:
`dash`, which is the default for ubuntu if my understanding is correct

Reply:
Ah weird. Ok I can override the `SHELL` then